src/test/rustdoc/primitive-generic-impl.rs

```diff
@@ -2,4 +2,4 @@
 
 include!("primitive/primitive-generic-impl.rs");
 
-// @has foo/primitive.i32.html '//div[@id="impl-ToString"]//h3[@class="code-header in-band"]' 'impl<T> ToString for T'
+// @has foo/primitive.i32.html '//div[@id="ToString-1"]//h3[@class="code-header in-band"]' 'impl<T> ToString for T'
```

Comment:
Why did this change? This seems like a bug in this PR.

Reply:
See https://github.com/rust-lang/rust/pull/89737#issuecomment-939505392 and https://github.com/rust-lang/rust/pull/89737#issuecomment-939596941

I will resolve the conflicts now and see if it's fixed.